docs/why.qmd

```diff
@@ -1,35 +1,53 @@
 
-## Seamless backend integration
+![Present state](./present.png)
 
-Ibis seamlessly integrates with multiple backend systems, such as BigQuery, Snowflake, MySQL, PostgreSQL, and more. This flexibility allows you to leverage the power of Ibis regardless of where your data resides. Whether you prefer working with SQL or pandas-like syntax, Ibis enables you to harness the capabilities of your chosen backend effortlessly.
+While portability with Ibis isn't perfect, commonalaties across backends and SQL dialects combined with years of engineering effort product a pretty good result. In the long-term future, we hope for standard Intermediary Representation (IR) formats for compute to simplify this further.
 
-## Efficient query execution
+![Future state](./future.png)
 
-Ibis optimizes your queries for efficient execution and performance. It leverages the strengths of each backend to ensure that your data operations are executed as quickly as possible. This optimization translates into faster query results, reducing the time it takes to derive insights from your data.
+## Ecosystem
 
-## Powerful analytics capabilities
+Ibis is part of a larger ecosystem of Python data tools. Ibis is designed to work well with other tools in the ecosystem, and we hope to make it easier to use Ibis with other tools over time. 
 
-Ibis empowers you to perform advanced analytics at scale. With built-in functions for statistical analysis, grouping, aggregation, and more, you can easily derive meaningful insights from your data. Whether you need to calculate descriptive statistics, perform regression analysis, or create custom analytics workflows, Ibis provides the tools you need.
+### Local experience
 
-## Easy integration with data science ecosystem
+By default, Ibis offers a great local experience for working with many file formats. DuckDB is the default backend, with Polars and DataFusion as two other great local options. Many of the backends can run locally but require more setup than a pip installation.
 
-As an integral part of the Python data science ecosystem, Ibis seamlessly integrates with popular libraries such as pandas, PySpark, Polars, and DataFusion. This integration allows you to combine the power of Ibis with the extensive capabilities of these libraries, enabling you to build end-to-end data pipelines and perform complex data analyses with ease.
+### Scaling up and out
 
-## Robust input and output support
+After prototyping on a local backend, some directly scale in the cloud. For instance, you can prototype on DuckDB and deploy with MotherDuck. You can scale from any Python client with Ibis installed to whatever your backend supports.
```

Comment:
```suggestion
After prototyping on a local backend, directly scale in the cloud.You can prototype on DuckDB and deploy with MotherDuck. You can scale from any Python client with Ibis installed to whatever your backend supports.
```

The last sentence could use some massaging -- I don't think it will necessarily be clear to a user what their backend supports re: scaling